cassandra/tests/common.py

```diff
@@ -0,0 +1,107 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.dev import get_docker_hostname, get_here
+
+CHECK_NAME = "cassandra"
+
+HERE = get_here()
+HOST = get_docker_hostname()
+
+"""
+# not all metrics will be available in our E2E environment
+# specifically rate and count metrics:
+    "cassandra.net.up_endpoint_count",
```

Comment:
I think that those four metrics are in the `metrics.yaml` file instead of the `conf.yaml` and that `load_jmx_config` doesn't take the `metrics.yaml` into account.

They should be taken into account

Reply:
I tried adding them to the `conf.yaml` and was still not able to collect them ( though I will still add them back in ). 
I think this is related to the JMX PR you mentioned earlier. Looks like Rate and count metrics aren't able to be collected unless you can run the check more than once.  

>For end to end testing, we use the list_with_metrics command. It only
does one iteration though, which means we don't have output for counters
and rates. This adds a new command running 2 iterations.

https://github.com/DataDog/jmxfetch/pull/267

Reply:
`cassandra.net.up_endpoint_count` and `cassandra.net.down_endpoint_count` should be gauges so this shouldn't matter.
I've been able to collect them with the `agent check` command when I merged the PR that includes those two so I think it should be doable to add them in the test.

Reply:
You're right, added again and it worked.  ( though does this now count as a change for the changelog? )